gomint-server/src/main/java/io/gomint/server/network/packet/PacketResourcePackStack.java

```diff
@@ -68,6 +70,7 @@ public void deserialize( PacketBuffer buffer, int protocolID ) {
         }
 
         this.experimental = buffer.readBoolean();
+        this.gameVersion = buffer.readString();
```

Comment:
Just don't store this, document what it is with a comment and throw away the result (simply remove the variable)